mixins/segment-mixin.ts

```diff
@@ -0,0 +1,57 @@
+declare global {
+  interface Window {
+    digitalData: any
+    _analytics: any,
+    bluemixAnalytics: any
+  }
+}
+
+export const segmentMixin = {
```

Comment:
We should provide an explanation of how to use this mixin since it needs, at least:
* To add the Bluemix Analytics script to `nuxt.config.js`
* To add the `belongsTo` field to the classes using the mixing.
* To add the `mixin` to the pages you want to track.

Please, also add the explanation of the semantics of `belongsTo` too.